### PR TITLE
[WiP] bump versions and fix json object issue in test

### PR DESCRIPTION
**Pull request by omegasoft7**

### CODE REVIEW

File: JsonAPI/build.gradle [hunk 1]
```diff
@@ -2,14 +2,14 @@ apply plugin: 'com.android.library'
...
-    buildToolsVersion '25.0.2'
+    buildToolsVersion '25.0.3'
     compileSdkVersion 25
 
     defaultConfig {
         minSdkVersion 9
         targetSdkVersion 25
-        versionCode 17
-        versionName "1.4.2"
+        versionCode 16
```
Comment (lekz112) [inline JsonAPI/build.gradle]: Shouldn't we increase version code instead?

Comment (bhurling) [inline JsonAPI/build.gradle]: Do we actually need versionCode and versionName for library projects? we set it a second time further down below. I think that one will be used to generate the artifact

File: JsonAPI/build.gradle [hunk 2]
```diff
@@ -2,14 +2,14 @@ apply plugin: 'com.android.library'
 apply plugin: 'com.novoda.bintray-release'
 
 android {
-    buildToolsVersion '25.0.2'
+    buildToolsVersion '25.0.3'
```
Comment (bhurling) [inline JsonAPI/build.gradle]: Guess we also need to bump the build tools version in `.travis.yml`

**Comment by omegasoft7** [comment]
@lekz112 @bhurling As this PR was only a failed experiment for us and we found another solution to mock JsonApi responses, I am going to remove this branch

**Pull request closed without merging**